pycbc/events/stat.py

```diff
@@ -1716,26 +1745,128 @@ def get_statistic(stat):
         raise RuntimeError('%s is not an available detection statistic' % stat)
 
 
-def get_sngl_statistic(stat):
+def insert_statistic_option_group(parser):
```

Comment:
to be consistent with my naming suggestion, maybe this should be insert_ranking_option_group